_posts/one-password.md

```diff
@@ -0,0 +1,75 @@
+
+### One last attempt
+
+LastPass comes with a desktop application. I tried that:
+
+1. stopped sharing the accounts.
+1. I signed in to the browser extension using my account (the one I use the most).
+1. I signed in to the desktop application with work credentials.
+
+It works, somewhat. The Windows desktop application SUCKS as if there is no tomorrow. It's a UWP application so ugly that one thinks it's a leftover from a developer test.
+
+Me: what is this UWP thing? Let me try that.
+LastPass: that's great. Let's go to production.
+
+It's horrid, and it's missing features. Unusable.
+
+## What's next?
+
+I planned for a future switch when I upgraded my account to premium. I set a reminder for me nine months ahead to start looking for alternatives with a significant enough buffer time before the LastPass renewal.
+
+When the time came, based on the early cost of a password management solution, I decided not to go through the pain of other password manager trials. I didn't want to update all my devices to new software with a trial license. Import all my passwords, play with the bits, and switch to a different trial.
+
+Based on friends' and colleagues' recommendations, I picked [1Password](https://1password.com) and went for a 1-year Personal subscription. I only checked two things:
+
+- The features listed on the website satisfied my requirements
+- There was support to import from LastPass
+
+### So what?
+
+I've been using 1Password for six months, and the balance is not that great.
+
+Is it better than LastPass? Yes and no. It depends.
+
+The biggest issue is GDPR related. By default, data are stored outside the European Union, posing a privacy concern. There is an option, at subscription time, to store data in the EU; however, the sign-up process doesn't highlight it enough (it's a euphemism). Afterward, it's possible to move data across regions, but the process is manual, cumbersome, and far from being user-friendly, which is never a good sign, in my opinion. I'll try it sooner or later.
+
+The second issue is 2FA-related. 1Password, as LastPass does, supports storing the two-factor authentication token in the password manager. It's a handy feature, nonetheless. However, my spider sense perceives it as a security risk. There are scenarios in which it might be a [good enough compromise](https://security.stackexchange.com/questions/194142/is-it-safe-to-store-2fa-tokens-together-with-passwords-in-1password#194279), and [others](https://polansky.co/blog/psa-2fa-password-managers/). So I decided to continue using [Authy](https://authy.com) as the 2FA token storage.
+
+And there comes the third issue. It seems impossible, or at least I couldn't find a way, to disable 2FA support in 1Password. That means 1Password will try to fill tokens when requested automatically. It's again a user experience problem and isn't very pleasant.
```

Comment:
I believe you can delete the "one-time password" field from individual logins and it won't fill in automatically so that it doesn't automatically fill in the authentication token. In 1Password itself, you'll still see a big purple "This site supports 2FA" message at the top which you have to ignore but it's fine for the regular workflow.

Reply:
Can you please post this as a comment too?